rfc/text/advanced/ap_10_rpc_progressive_call_invocations.md

```diff
@@ -1,25 +1,25 @@
-## Progressive Calls {#rpc-progressive-calls}
+## Progressive Call Invocations {#rpc-progressive-call-invocations}
 
-A *Caller* may issue a progressive call. This can be useful in a few cases:
+A *Caller* may issue a progressive call with progressive call invocations. This can be useful in a few cases:
 
 * Payload is too big to send it whole in one request, e.g., uploading a file.
 * Long-term data transfer that needs to be consumed early, such as a media stream.
 * RPC is called too often and overall processing can be optimized: avoid the need to generate a new id for requests,
   initiate data structures for a new call, etc.
 
-In such cases, a procedure implemented by a *Callee* and registered at a *Dealer* may be made to receive progressive calls,
-where the *Callee* waits after receiving the entire set of payload chunks before sending the result. 
+In such cases, a procedure implemented by a *Callee* and registered at a *Dealer* may be made to receive progressive call invocations,
+where the *Callee* waits after receiving the entire set of payload chunks before sending the result.
```

Comment:
Looking at this sentence, I start thinking that we should rewrite it. As `waits after receiving the entire set of payload` is not the only possible way. And *Callee* may start processing data ASAP. And even return chunks of results (with the help of progressive call results)

Reply:
We are describing the Progressive Call Invocations feature independently of Progressive Call Results at this point. The new *Progressive Calls with Progressive Call Results* subsection I consolidated discussed the mixing of both.

Reply:
what is a "payload chunk"? this term isn't defined or required